k8s/volumes.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: jasminegraph-metadb
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 200Mi
+  accessModes:
+    - ReadWriteOnce
+  claimRef:
+    namespace: default
+    name: jasminegraph-metadb-claim
+  hostPath:
+    path: "${metadb_path}"
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: jasminegraph-metadb-claim
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 200Mi
+
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: jasminegraph-performancedb
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 200Mi
```

Comment:
What ever the amount we allocated it will be exceeded at a point.
So, I think we need to have a mechanism to clear the past data periodically?.

Reply:
What ever the amount we allocated it will be exceeded at a point.
So, I think we need to have a mechanism to clear the past data periodically?.

Reply:
Rather than clearing is there a way we can have some warning mechanism so that we can warn when the file size exceeds some threshold? Automated clearing may delete some wanted data as well. Based on the warning message system admin can decide what to do.

Reply:
As per our offline discussion lets get the relevant monitoring chart implemented in our Grafaana dashboard so that we can monitor the disk usage